platforms/quorum/configuration/roles/create/certificates/ambassador/tasks/main.yaml

```diff
@@ -9,6 +9,8 @@
 - name: Create Ambassador certificates
   include_tasks: nested_main.yaml
   vars:
+    name: "{{ org.name }}"
```

Comment:
Please fix this. I don't understand why do you need 3 variables just to define an organization name.
There are 2 variables already defined in the calling task (present in deploy-network.yaml) namely `organization` and `org_name` doing the exact same thing.